### DISCUSSION TITLE
Second DB9-Joystick

### DISCUSSION THREAD

**Discussion by sgrockfort** [vossstef/tang_nano_20k_c64#101] [General]
Hello, 

I would be interested to know whether it's generally possible to connect a second DB9 joystick in a future release. 
Possibly also by replacing the dualshock. I understand that the existence of two connection variants benefit more users. If possible, an additional DB9 connection while retaining the dualshock would of course be the best way.  
A few colleagues and I want to have a PCB circuit board made and possibly print a case for it. In this context, the question arose as to whether a second DB9 connector could be planned directly for possible future use.
We are therefore interested in a general assessment of the feasibility. Firstly, irrespective of whether there is a roadmap for this or not. :-)

Thanks
Michael

**Comment by harbaum** [comment]
This is actually rather easy and only needs a modified top level, a custom cst pin mapping and a matching gprj project file. All three files can be derived from the original ones with minor changes.

This is indeed a perfect project for someone who's new to FPGA coding.

**Reply by sgrockfort** [reply]
ohh, wow, to be honest I didn't expect this.. I thought it would be a big change.  But it's good news of course :-) :-) 
Indeed, more or less I'm fully new with the fpga topic but very interested ... I'll try to figure out a solution..

Thanks alot
Michael
